metricbeat/metricbeat.template-es2x.json

```diff
@@ -287,29 +287,32 @@
         "system-filesystem": {
           "properties": {
             "avail": {
-              "type": "integer"
+              "type": "long"
```

Comment:
Probably going to conflict with this one (outcome should be mostly the same): https://github.com/elastic/beats/pull/1572